cypress/e2e/pages/cytAssist.cy.ts

```diff
@@ -342,16 +342,15 @@ describe('CytAssist Page', () => {
             selectSlideCostings('Faculty');
             enterLOTNumber();
             enterProbeLOTNumber();
-            cy.screenshot();
             cy.findByTestId('external-barcode').within(() => {
-              cy.findByRole('textbox').clear().type('H1-9D8VN2V{enter}').blur();
```

Comment:
Are all these waits necessary for the tests to pass, or were they put in for debugging?

Reply:
adding the waits stops the cytAssist tests from randomly failing. Adding the screenshot for debugging purposes  makes the tests pass, as the system pauses for sometime to take the screenshot.